site/docs/contribute.md

```diff
@@ -45,6 +45,17 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. For larger changes or additions to public
```

Comment:
What about objections? To me it feels like people tend to regard a GH "request changes" review as a -1 in ASF terms, so those should probably be accompanied by a technical justification according to ASF [guidelines](https://www.apache.org/foundation/voting.html#Veto).

Since most PR reviews happen in GitHub, would it be reasonable to explicitly link the GH "changes requested" option to ASF -1 Vetoes?

Reply:
IMO -1 votes are more severe, I'll add some more content here.

Reply:
@dimas-b updated the proposal, let me know if this make sense.

Reply:
SGTM. Thx!